nuxeo-core/nuxeo-core-io/src/main/java/org/nuxeo/ecm/core/io/avro/BlobPropertyMapper.java

```diff
@@ -62,12 +62,12 @@ public Object fromAvro(Schema schema, Object input) {
         case RECORD:
             try {
                 GenericRecord record = (GenericRecord) input;
-                String mimeType = (String) record.get(service.encodeName("mime-type"));
-                String encoding = (String) record.get("encoding");
-                byte[] bytes = ((ByteBuffer) record.get("data")).array();
+                String mimeType = (String) record.get(service.encodeName(Constants.MIME_TYPE));
```

Comment:
you can import in a static way all constants you use, especially as `Constants` is pacakge protected.

Reply:
why would I static import all constants I use once ? I can't see any win

Reply:
It makes for more concise code at the point where it's used.